examples/Datasources.ipynb

```diff
@@ -0,0 +1,155 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Datasources"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append('/path/to/raster-foundry-python-client')"
```

Comment:
For user's who've `pip install`ed, this won't be necessary, since pip will have put the library in their path. For users in development, if they've done `scripts/update` and are in the resulting virtualenv, it also won't be necessary, since that script installs `rasterfoundry` into the virtualenv. So this cell can be removed.